documentation/alerts.html

```diff
@@ -98,52 +107,109 @@ <h5 class="mb-4">COMPONENT DEMO</h5>
-                        <i class="far fa-times-circle fa-2x close-icon text-cursor-pointer"></i>
+                        <span>
+                            <i class="fas fa-check px-3"></i>
+                            This is a success alert
+                        </span>
+                        <i class="far fa-times-circle fa-2x close-icon
+                        text-cursor-pointer"></i>
                     </div>
+
+                    <!-- Warning dismissable alert component with icon -->
                     <div class="alert-dismissable alert-warning p-4">
-                        <span><i class="fas fa-exclamation px-3"></i>This is a warning alert</span>
-                        <i class="far fa-times-circle fa-2x close-icon text-cursor-pointer"></i>
+                        <span>
+                            <i class="fas fa-exclamation px-3"></i>
+                            This is a warning alert
+                        </span>
+                        <i class="far fa-times-circle fa-2x close-icon
+                        text-cursor-pointer"></i>
                     </div>
+
+                    <!-- Error dismissable alert with icon -->
                     <div class="alert-dismissable alert-error p-4">
-                        <span><i class="fas fa-exclamation-triangle px-3"></i>This is an error alert</span>
-                        <i class="far fa-times-circle fa-2x close-icon text-cursor-pointer"></i>
+                        <span>
+                            <i class="fas fa-exclamation-triangle px-3"></i>
+                            This is an error alert
+                        </span>
+                        <i class="far fa-times-circle fa-2x close-icon
+                        text-cursor-pointer"></i>
                     </div>
+
+                    <!-- Info dismissable alert with icon -->
                     <div class="alert-dismissable alert-info p-4">
-                        <span><i class="fas fa-info-circle px-3"></i>This is an info alert</span>
-                        <i class="far fa-times-circle fa-2x close-icon text-cursor-pointer"></i>
+                        <span>
+                            <i class="fas fa-info-circle px-3"></i>
+                            This is an info alert
+                        </span>
+                        <i class="far fa-times-circle fa-2x close-icon 
+                        text-cursor-pointer"></i>
                     </div>
             
                 </div>
                 <div class="code-demonstration">
                     <script src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2Fdev-enforced%2FGentle_UI%2Fblob%2F62eee2a0e55d05d594667386896fe5a6aee0c964%2Fcomponents%2Falert%2Falert.html%23L56-L75&style=atom-one-dark-reasonable&showBorder=on&showCopy=on"></script>
                 </div>
+                <p class="txt-sm">JS CODE FOR DISMISSABLE ALERTS:</p>
```

Comment:
also is there a need to provide the js code here, may be just providing the import link would work, the user won't need the code behind it, he will be needing working js functionality, i could be wrong here(correct me if i am wrong)